modules/utxo-lib/src/inscriptions.ts

```diff
@@ -0,0 +1,66 @@
+import * as assert from 'assert';
+import { Payment, script as bscript } from 'bitcoinjs-lib';
+import { p2tr } from './payments';
+import { ecc as eccLib } from './';
+
+const OPS = bscript.OPS;
+const MAX_LENGTH_TAP_DATA_PUSH = 520;
+
+/**
+ * The max size of an individual OP_PUSH in a Taproot script is 520 bytes. This
+ * function splits inscriptionData into an array buffer of 520 bytes length.
+ * https://docs.ordinals.com/inscriptions.html
+ * @param inscriptionData
+ */
+function sliceInscriptionData(inscriptionData: Uint8Array) {
```

Comment:
We typically use `Buffer` (we are even using it here)

There is nothing really inscription-specific here. Let's just call it `splitBuffer`. This makes the variable names a bit simpler as well.

```suggestion
function splitBuffer(buf: Buffer, chunkSize: number) {
```